Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/AuthMethodManager.java

```diff
@@ -171,13 +179,25 @@ private synchronized AzureManager getAzureManager(final AuthMethod authMethod) t
         return localAzureManagerRef;
     }
 
+    public BaseADAuthManager getAdAuthManagerBy(final AuthMethod authMethod) {
```

Comment:
change to `getAdAuthManagerByAuthMethod`?

Reply:
Fix it!